index.bs

```diff
@@ -2124,6 +2124,50 @@ document loading at which a navigation command will return.
 
 ### Commands ### {#module-browsingContext-commands}
 
+#### The browsingContext.bringToFront Command ####  {#command-browsingContext-bringToFront}
```

Comment:
I think focusing on an element can be done via `script` evaluation so I am not sure if we would need to add it. At the same time, I feel like the name `focus` would not reflect that it also does activate the tab making it visible. Happy to rename to `focus` or `activate` though if you think this is a better name.

Reply:
I for myself would vote for `activate` given that this better reflects the activation of the tab.

Do we need special steps to ensure that the window itself is in the foreground (top-most) in case remote ends cannot correctly handle the focus when a window is hidden behind another one? In Firefox this is not a problem, but I wonder about other implementations.

Reply:
I have no objections with `activate()`.

Reply:
+1 for `activate`.

Reply:
+1 to `activate()`